src/bindings/python/pyproject.toml

```diff
@@ -1,3 +1,45 @@
-[tool.black]
-line-length = 160
-include = '\.pyi?$'
+[project]
```

Comment:
Shouldn't pyproject.toml placed to the root of the repo to cover also tools/benchmark_app?

Reply:
That is discussible, I thought about that too, but at this case we would need to put in the root setup.py too until we drop it at all

Reply:
agree about moving to the root.